Tools/scripts/build_options.py

```diff
@@ -110,6 +110,7 @@ def __init__(self,
     Feature('OSD', 'PLUSCODE', 'HAL_PLUSCODE_ENABLE', 'Enable PlusCode', 0, 'OSD'),
     Feature('OSD', 'OSD_PARAM', 'OSD_PARAM_ENABLED', 'Enable OSD param', 0, 'OSD'),
     Feature('OSD', 'OSD_SIDEBARS', 'HAL_OSD_SIDEBAR_ENABLE', 'Enable Scrolling Sidebars', 0, 'OSD'),
+    Feature('OSD', 'OSD_CRSF_LINK_STATS', 'AP_OSD_CRSF_EXTENSIONS_ENABLED', 'Enable OSD panels for CRSF expanded link stats data', 0, "OSD,RC_CRSF"),
```

Comment:
need the PANELS define as well I think

Reply:
Since the PANELS define is conditionally defined [here](https://github.com/ArduPilot/ardupilot/pull/25844/files#diff-2d27ad6b9c73dbb0c2e46daf08ef2b34117a0665902d9115d1185d53e61e52aeR55), as explained in the previous reply, I don't think it's necessary to add it to build_options, as it's supposed to be "local" define to the AP_OSD code.

Reply:
if you provide an ability to exclude something, you need to be able to selectively build it if its removed...otherwise remove the exclusion if it will never be used

Reply:
> if you provide an ability to exclude something, you need to be able to selectively build it if its removed...otherwise remove the exclusion if it will never be used

Check AP_OSD.h, line 55 [here](https://github.com/ArduPilot/ardupilot/pull/25844/files#diff-2d27ad6b9c73dbb0c2e46daf08ef2b34117a0665902d9115d1185d53e61e52aeR55), there you will see what I mean.